packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -35,6 +35,7 @@ export const DEFAULT_GET_MARKET_ORDERS_OPTS: GetMarketOrdersOpts = {
     gasSchedule: {},
     allowFallback: true,
     shouldBatchBridgeOrders: true,
+    rfqt: undefined,
```

Comment:
Is this change actually required? I would expect us not having to populate this key with `undefined` since it's defined as `rfqt?` https://github.com/0xProject/0x-monorepo/blob/6c62c3342ca21c89f6e3da527d0b0b83e4040033/packages/asset-swapper/src/utils/market_operation_utils/types.ts#L192

Reply:
It's not required, thank you for noticing it. I think I hadn't yet made it optional in a previous iteration of hacking on it.  I got rid of this.